fluid/CMakeLists.txt

```diff
@@ -12,18 +15,23 @@ file(GLOB_RECURSE PADDLE_MOBILE_H src/*.h)
 # include headers
 include_directories(src/)
 
+
+include_directories(/usr/local/Cellar/openblas/0.2.20_2/include/)
+link_directories(/usr/local/Cellar/openblas/0.2.20_2/lib/)
```

Comment:
Do not add a 3rd-party package like this. You can use `ExternalProject` in `cmake`.